ck

```diff
@@ -55,7 +55,7 @@ then
 else
   if [[ -n $KUBECONFIG ]]
   then
-    verbose -1 "WARNING: KUBECONFIG is set and is being ignored by $(basename ${0})"
+    verbose -1 "WARNING: KUBECONFIG is set and is being ignored by $CLUSTERPOOL_CONTEXT_NAME"
```

Comment:
This is meant to be the name of the command, and not the name of the context the command uses. Use a soft link rather than an alias. For example:
```
ln -s /Users/kcormier/ocm/cluster-keeper/ck ck_other
% ls -l ck*
lrwxr-xr-x  1 kcormier  admin  37 24 Feb 14:10 ck -> /Users/kcormier/ocm/cluster-keeper/ck
lrwxr-xr-x  1 kcormier  admin  37 24 Mar 12:23 ck_other -> /Users/kcormier/ocm/cluster-keeper/ck
% export KUBECONFIG=fake                              
% ck list cc
WARNING: KUBECONFIG is set and is being ignored by ck
...
% ck_other list cc
WARNING: KUBECONFIG is set and is being ignored by ck_other
```